restcomm/restcomm.telephony/src/main/java/org/restcomm/connect/telephony/CallManager.java

```diff
@@ -649,6 +653,134 @@ private void info(final SipServletRequest request) throws IOException {
         }
     }
 
+    private void transfer(SipServletRequest request) throws Exception {
+        final SipApplicationSession appSession = request.getApplicationSession();
+        ActorRef call = (ActorRef) appSession.getAttribute(Call.class.getName());
+
+        final Timeout expires = new Timeout(Duration.create(60, TimeUnit.SECONDS));
+
+        Future<Object> infoFuture = (Future<Object>) ask(call, new GetCallInfo(), expires);
+        CallResponse<CallInfo> infoResponse = (CallResponse<CallInfo>) Await.result(infoFuture,
+                Duration.create(10, TimeUnit.SECONDS));
+        CallInfo callInfo = infoResponse.get();
+
+        CallDetailRecordsDao dao = storage.getCallDetailRecordsDao();
+
+        CallDetailRecord cdr = dao.getCallDetailRecord(callInfo.sid());
+        Sid parentCallSid = cdr.getParentCallSid();
+        cdr = dao.getCallDetailRecord(parentCallSid);
+
+        call = lookup(new GetCall(cdr.getCallPath()));
+        IncomingPhoneNumber number = storage.getIncomingPhoneNumbersDao().getIncomingPhoneNumber(cdr.getTo());
+
+        // Get first call leg observers
+        Future<Object> future = (Future<Object>) ask(call, new GetCallObservers(), expires);
+        CallResponse<List<ActorRef>> response = (CallResponse<List<ActorRef>>) Await.result(future,
+                Duration.create(10, TimeUnit.SECONDS));
+        List<ActorRef> callObservers = response.get();
+
+        // Get the Voice Interpreter currently handling the call
+        ActorRef existingInterpreter = callObservers.iterator().next();
+
+        // Get the outbound leg of this call
+        future = (Future<Object>) ask(existingInterpreter, new GetRelatedCall(call), expires);
+        Object answer = (Object) Await.result(future, Duration.create(10, TimeUnit.SECONDS));
+
+        ActorRef relatedCall = null;
+        List<ActorRef> listOfRelatedCalls = null;
+        if (answer instanceof ActorRef) {
+            relatedCall = (ActorRef) answer;
+        } else if (answer instanceof List) {
+            listOfRelatedCalls = (List<ActorRef>) answer;
+        }
+
+        if(logger.isInfoEnabled()) {
```

Comment:
@vmykhailiuk better change these log statements to match the transfer operation here instead of LCM